index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Loan System</title>
+    <script src="https://kit.fontawesome.com/60f96566f6.js" crossorigin="anonymous"></script>
+    
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="top-bar">
+    <ul class="top-bar-items">
+      <li class="top-bar-icon"><i class="fab fa-facebook-f"></i></li>
+      <li class="top-bar-icon"><i class="fab fa-twitter"></i></li>
+      <li class="top-bar-icon"><a href="#">english</a></li>
+      <li class="top-bar-icon"><a href="#">my page</a></li>
+      <li class="top-bar-icon">logout</li>
+    </ul>
+  </div>
+    <nav>
+        <div class="logo"><a href="index.html">My Logo</a></div>
+        <ul class="navbar">
+            <li class="nav-items"><a href="index.html">home</a></li>
+            <li class="nav-items"><a href="about.html">about</a></li>
+            <i class="fas fa-bars" onclick="displayMenu()"></i>   
+        </ul>
+        
+    </nav>
+    <div id='popup'></div>
+    <header>
+        <h5 class="first-text">"Hello! Dear Businessman"</h5>
+        <h1 class="second-text">We have the softare you need</h1>
+        <p class="paragraph">We have develop the system you were looking for, wich will help you
+            track your loans, payments and reports with all the useful data you need.
+        </p>
+    </header>
```

Comment:
- [ ] This header design is still missing from both laptop and mobile view. Also, in mobile view, there are a lot of whitespaces. Can you see the difference?

### Your design


![Loan_System](https://user-images.githubusercontent.com/22434660/124183650-2693a980-dad6-11eb-85b6-73e7a452a0c9.png)

### Template design

![Screenshot 2021-07-02 at 01-34-11 CC Global Summit 2015](https://user-images.githubusercontent.com/22434660/124183628-209dc880-dad6-11eb-8b5f-0ca7e724f420.png)

Reply:
hello, i dont know what is happening in my computer and cellphone all seems alright, bellow you can see my cellphone screenshot
index
 ![Alt text](https://github.com/elassis/capstone_module_1/blob/home_about_branch/images/mobileshoot-1.jpeg "index top")

about
![Alt text](https://github.com/elassis/capstone_module_1/blob/home_about_branch/images/mobileshoot-2.jpeg "index top")

Reply:
and also about the design, the documentation doesn't say that we have to make the project exactly equals since we don't have all those shapes and images at our disposal

Reply:
- I got your points but what I saw a lot of blank space in the mobile view so, I thought it's better to put element that makes your website more pretty and similar to design. Hope next reviewer will not see those blank space in their system. If so, maybe they confirm with the above screenshot you attached.